tests/reducers/medication-reducers.test.js

```diff
@@ -330,4 +330,57 @@ describe('Medication Reducers', () => {
       expect(reducer(state, action)).toEqual(state);
     });
   });
+
+  describe('createFhirResource', () => {
+    it('creates MedicationOrder for DSTU2', () => {
+      const patientId = 'patient-123';
+      const fhirVersion = '1.0.3';
```

Comment:
nit: let's use 1.0.2 as the official dstu2 version (http://www.hl7.org/fhir/directory.cfml)

Reply:
Done.